confluent_kafka/src/confluent_kafka.c

```diff
@@ -296,26 +327,12 @@ static PyTypeObject KafkaErrorType = {
 	0,                         /* tp_descr_get */
 	0,                         /* tp_descr_set */
 	0,                         /* tp_dictoffset */
-	0,                         /* tp_init */
-	0                          /* tp_alloc */
+	KafkaError_init0,          /* tp_init */
+	0,                         /* tp_alloc */
+        KafkaError_new             /* tp_new */
```

Comment:
Add unit tests in tests/test_Error.py (or whatever it is called) for this new functionality.

Reply:
Sure we get coverage from the negatice serializer test but we can add one specific to KafkaError